lib/node_modules/@stdlib/repl/test/integration/test.auto_close_pairs.js

```diff
@@ -284,6 +290,154 @@ function assertNoAutoClose( t, fixture, done ) {
 	}
 }
 
+/**
+* Asserts that a provided expression triggers expected automatic deletion of closed symbols.
+*
+* @private
+* @param {Object} t - test object
+* @param {Object} fixture - fixture object
+* @param {Function} done - callback to invoke upon completion
+*/
+function assertAutoDelete( t, fixture, done ) {
+	var expected;
+	var istream;
+	var opts;
+	var N;
+	var r;
+
+	istream = new DebugStream({
+		'name': 'repl-input-stream'
+	});
+	opts = {
+		'input': istream,
+		'inputPrompt': '',
+		'quiet': true,
+		'settings': defaultSettings()
+	};
+	r = repl( opts, onClose );
+
+	istream = new DebugStream({
+		'name': 'repl-input-stream'
+	});
+	opts = {
+		'input': istream,
+		'inputPrompt': '',
+		'quiet': true,
+		'settings': defaultSettings()
+	};
+	r = repl( opts, onClose );
+
+	// Construct the expected output:
+	expected = fixture.expression.substring( 0, fixture.cursor-1 );
+	expected += fixture.expression.substring( fixture.cursor+1 );
+
+	// Emulate the presence of an existing expression:
+	istream.write( fixture.expression );
+
+	// Move the cursor to where we want to delete a character:
+	N = fixture.expression.length - fixture.cursor;
+	moveBack( istream, N );
+
+	// Enable auto-close:
+	r.settings( 'autoClosePairs', true );
+
+	// Delete the character in order to trigger auto-delete:
+	r._rli.write( null, {  // eslint-disable-line no-underscore-dangle
```

Comment:
Did you try using an ANSI escape code written to `istream`? In general, we should avoid the assumption of private properties in tests and only test against public APIs.